scheduler/cloudformation/cloudformation.go

```diff
@@ -274,9 +277,138 @@ func (s *Scheduler) submit(ctx context.Context, tx *sql.Tx, app *scheduler.App,
 		}
 	}
 
+	go s.waitForDeploymentToStabilize(ctx, app, stackName, events, done)
```

Comment:
I think the correct place to do this, would be immediately after [this call](https://github.com/mhahn/empire/blob/c1ea348005d0c4eb832858810a87cf7926295118/scheduler/cloudformation/cloudformation.go#L572), since the stack operation lock is held, and the stack has been updated/created successfully at that point.